### DISCUSSION TITLE
Support For Stadia ?

### DISCUSSION THREAD

**Discussion by Kaecls** [ndeadly/MissionControl#229] [Ideas]
Is it possible to add support for google stadia ?

**Comment by ndeadly** [comment]
The Stadia controller uses WiFi Direct (I think) and doesn't currently support Bluetooth. I've read it may be possible to enable it via a firmware update in the future but I don't believe that's happened yet. If they enable Bluetooth support I don't see why it shouldn't be possible to add it to Mission Control.

**Reply by Kaecls** [reply]
Ohh okay thanks , how about using it with usb does that work with mission controll ?

**Reply by ndeadly** [reply]
USB is not currently supported. I have plans to add it in the future, but I want to do it differently to sys-con and mitm the `usb` service like I do with Bluetooth. This would allow for controllers to support rumble, motion, button remapping etc which is not possible with sys-con. Unfortunately, the tool to capture the IPC calls doesn't currently work for `usb` so support is on hold until that can be figured out.

**Reply by ndeadly** [reply]
Good news, Google is releasing Bluetooth support for the Stadia controller soon!. Let's hope it's not using Bluetooth LE

https://community.stadia.com/t5/Stadia-General/A-Gift-from-the-Stadia-Team-amp-Bluetooth-Controller/m-p/85936
